### PR TITLE
IPv6 LAN config spec

### PR DESCRIPTION
**Pull request by lamontj**
Add IPv6 lan configuration parameters to the spec include file.  Add read-only support for IPv6 parameters to the API.

### CODE REVIEW

**Comment by chu11** [comment]
Overall looks good, need to review details again before merging.  But could you add the appropriate template documentation into ```libfreeipmi/include/freeipmi/templates/```.  Basically it's a cut & paste, w/ some search & replace, and some hand mangling to make the documentation pretty.  Thanks.

File: libfreeipmi/include/freeipmi/spec/ipmi-lan-configuration-parameters-spec.h [hunk 1]
```diff
@@ -23,40 +23,71 @@
...
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_IPV4_SUPPORT                                          50
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_IPV4_ADDRESSING_ENABLES                               51
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_HEADER_STATIC_TRAFFIC_CLASS                           52
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_HEADER_STATIC_HOP_LIMIT                               53
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_HEADER_FLOW_LABEL                                     54
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_STATUS                                                55
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_STATIC_ADDRESSES                                      56
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_DHCPV6_STATIC_DUID_STORAGE_LENGTH                     57
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_DHCPV6_STATIC_DUIDS                                   58
+#define IPMI_LAN_CONFIGURATION_PARAMETER_IPV6_DYNAMIC_ADDRESSES                                     59
```
Comment (chu11) [inline libfreeipmi/include/freeipmi/spec/ipmi-lan-configuration-parameters-spec.h]: should be "address", not plural

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 2]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 8, "set_selector", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 8, "block_selector", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 128, "duid", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_addresses_rs =
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: like prior comment, "address", not addresses

File: libfreeipmi/include/freeipmi/cmds/ipmi-lan-cmds.h [hunk 3]
```diff
@@ -220,6 +256,37 @@ extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_rmcpplus_messag
...
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_ipv4_support_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_ipv4_addressing_enables_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_header_static_traffic_class_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_header_static_hop_limit_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_header_flow_label_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_status_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_addresses_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dhcpv6_static_duid_storage_length_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dhcpv6_static_duids_rs;
+extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_addresses_rs;
```
Comment (chu11) [inline libfreeipmi/include/freeipmi/cmds/ipmi-lan-cmds.h]: not plural

File: libfreeipmi/api/ipmi-lan-cmds-api.c [hunk 4]
```diff
@@ -1477,7 +1477,7 @@ ipmi_cmd_set_lan_configuration_parameters_bad_password_threshold (ipmi_ctx_t ctx
       API_SET_ERRNUM (ctx, IPMI_ERR_PARAMETERS);
       return (-1);
     }
-
```
Comment (chu11) [inline libfreeipmi/api/ipmi-lan-cmds-api.c]: errant whitespace change

File: libfreeipmi/api/ipmi-lan-cmds-api.c [hunk 5]
```diff
@@ -1798,7 +1798,7 @@ ipmi_cmd_get_lan_configuration_parameters_ipv4_header_parameters (ipmi_ctx_t ctx
       ERR_TRACE (ipmi_ctx_errormsg (ctx), ipmi_ctx_errnum (ctx));
       return (-1);
     }
-  
+
```
Comment (chu11) [inline libfreeipmi/api/ipmi-lan-cmds-api.c]: errant whitespace change

File: libfreeipmi/api/ipmi-lan-cmds-api.c [hunk 6]
```diff
@@ -2234,6 +2234,750 @@ ipmi_cmd_get_lan_configuration_parameters_bad_password_threshold (ipmi_ctx_t ctx
...
+    {
+      ERR_TRACE (ipmi_ctx_errormsg (ctx), ipmi_ctx_errnum (ctx));
+      return (-1);
+    }
+
+  return (0);
+}
+
+int
+ipmi_cmd_get_lan_configuration_parameters_ipv6_static_addresses (ipmi_ctx_t ctx,
```
Comment (chu11) [inline libfreeipmi/api/ipmi-lan-cmds-api.c]: not plural

File: libfreeipmi/include/freeipmi/api/ipmi-lan-cmds-api.h [hunk 7]
```diff
@@ -397,6 +397,223 @@ int ipmi_cmd_get_lan_configuration_parameters_bad_password_threshold (ipmi_ctx_t
...
+                                                                                      fiid_obj_t obj_cmd_rs);
+
+int ipmi_cmd_get_lan_configuration_parameters_ipv6_dhcpv6_static_duids (ipmi_ctx_t ctx,
+                                                                        uint8_t channel_number,
+                                                                        uint8_t get_parameter,
+                                                                        uint8_t set_selector,
+                                                                        uint8_t block_selector,
+                                                                        fiid_obj_t obj_cmd_rs);
+
+int ipmi_cmd_get_lan_configuration_parameters_ipv6_dynamic_addresses (ipmi_ctx_t ctx,
```
Comment (chu11) [inline libfreeipmi/include/freeipmi/api/ipmi-lan-cmds-api.h]: not plural

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 8]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+    { 0, "", 0}
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_1_ip_address_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data for ip?

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 9]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_1_mac_address_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing mac data?

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 10]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_1_prefix_length_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 11]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_1_prefix_value_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 12]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_2_ip_address_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 13]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_2_mac_address_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 14]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_2_prefix_length_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 15]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_static_router_2_prefix_value_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 16]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_number_of_dynamic_router_info_sets_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 17]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_router_info_ip_address_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 128, "router_ip_address", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing set selector

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 18]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_router_info_mac_address_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 48, "router_mac_address", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing set selector

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 19]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_router_info_prefix_length_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 8, "prefix_length", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing set selector

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 20]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_router_info_prefix_value_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 128, "prefix_value", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing set selector

File: libfreeipmi/cmds/ipmi-lan-cmds.c [hunk 21]
```diff
@@ -756,6 +756,335 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
...
+    { 0, "", 0}
+  };
+
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_dynamic_router_received_hop_limit_rs =
+  {
+    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
+    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
+    { 0, "", 0}
```
Comment (chu11) [inline libfreeipmi/cmds/ipmi-lan-cmds.c]: missing data?

**Comment by lamontj** [comment]
Incorporated the feedback.  I'm going to go back and re-review the responses in the spec vs the code, to see if there are any more that are missing.

File: libfreeipmi/api/ipmi-lan-cmds-api.c (hunk 6 above)
Comment (lamontj) [inline libfreeipmi/api/ipmi-lan-cmds-api.c]: Actually, command 56 is "IPv6 Static Addresses"

**Comment by lamontj** [comment]
I've added the missing set_selectors, so I think that's all of the review feedback addressed.  Today will be the slow walk through the spec to see if I can catch any more missing bits.

File: libfreeipmi/api/ipmi-lan-cmds-api.c (hunk 6 above)
Comment (chu11) [inline libfreeipmi/api/ipmi-lan-cmds-api.c]: ahh, you're right.

**Comment by chu11** [comment]
Awesome,  LMK when you've finished your review and then I'll take 1 more last skim.  Then hopefully we can merge soon.

**Comment by lamontj** [comment]
Review complete.  Looks good to me.  Over to you for that one last skim.

**Comment by chu11** [comment]
Awesome, looks good to me.  I squashed your changes into fewer commits then manually merged into master.  Thanks!

**Pull request closed without merging**